From 1f8cb31b07b4a3a0b1db7414d4e959c2812041ce Mon Sep 17 00:00:00 2001
From: Gareth Williams <gareth.williams.jx@renesas.com>
Date: Wed, 24 Feb 2021 16:27:03 +0000
Subject: [PATCH] Add support for building arm and aarch64 CL with Scons

Recreated for arm_compute v20.11

Signed-off-by: Jianming Qiao <jianming.qiao@bp.renesas.com>
Signed-off-by: Gareth Williams <gareth.williams.jx@renesas.com>
---
 SConstruct | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/SConstruct b/SConstruct
index 3b2be1176..0da030d19 100644
--- a/SConstruct
+++ b/SConstruct
@@ -194,11 +194,11 @@ if 'v7a' in env['estate'] and env['estate'] == '64':
 # Add architecture specific flags
 prefix = ""
 if 'v7a' in env['arch']:
-    env.Append(CXXFLAGS = ['-march=armv7-a', '-mthumb', '-mfpu=neon'])
+    env.Append(CXXFLAGS = ['-mthumb', '-mfpu=neon'])
     if env['os'] == 'android' or env['os'] == 'tizen':
         env.Append(CXXFLAGS = ['-mfloat-abi=softfp'])
     else:
-        env.Append(CXXFLAGS = ['-mfloat-abi=hard'])
+        env.Append(CXXFLAGS = ['-mfloat-abi=hard','-fPIC'])
 elif 'v8' in env['arch']:
     if 'sve' in env['arch']:
         env.Append(CXXFLAGS = ['-march=armv8.2-a+sve+fp16+dotprod'])
@@ -225,9 +225,7 @@ elif 'x86' in env['arch']:
 prefix = ""
 if 'x86' not in env['arch']:
     if env['estate'] == '32':
-        if env['os'] == 'linux':
-            prefix = "arm-linux-gnueabihf-" if 'v7' in env['arch'] else "armv8l-linux-gnueabihf-"
-        elif env['os'] == 'bare_metal':
+        if env['os'] == 'bare_metal':
             prefix = "arm-eabi-"
         elif env['os'] == 'android':
             prefix = "arm-linux-androideabi-"
@@ -235,7 +233,7 @@ if 'x86' not in env['arch']:
             prefix = "armv7l-tizen-linux-gnueabi-"
     elif env['estate'] == '64' and 'v8' in env['arch']:
         if env['os'] == 'linux':
-            prefix = "aarch64-linux-gnu-"
+            env.Append(CXXFLAGS = ['-fPIC'])
         elif env['os'] == 'bare_metal':
             prefix = "aarch64-elf-"
         elif env['os'] == 'android':
-- 
2.25.1

