From 40658c6224f2ac5235e8e951fa2167f590728fc6 Mon Sep 17 00:00:00 2001
From: Jianming Qiao <jianming.qiao@bp.renesas.com>
Date: Fri, 15 Jun 2018 11:45:50 +0100
Subject: [PATCH 2/2] Always use third_party/eigen

Signed-off-by: Jianming Qiao <jianming.qiao@bp.renesas.com>
Signed-off-by: Fabrizio Castro <fabrizio.castro@bp.renesas.com>
---
 cmake/Dependencies.cmake | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/cmake/Dependencies.cmake b/cmake/Dependencies.cmake
index 724ec52..8b4da7d 100644
--- a/cmake/Dependencies.cmake
+++ b/cmake/Dependencies.cmake
@@ -199,12 +199,7 @@ endif()
 
 # ---[ EIGEN
 add_definitions(-DEIGEN_MPL2_ONLY)
-find_package(Eigen3 QUIET)
-if(EIGEN3_FOUND)
-  include_directories(SYSTEM ${EIGEN3_INCLUDE_DIRS})
-else()
-  include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/third_party/eigen)
-endif()
+include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/third_party/eigen)
 
 # ---[ Python + Numpy
 if(BUILD_PYTHON)
-- 
2.7.4

