diff --git a/tensorflow/core/lib/gtl/inlined_vector.h b/tensorflow/core/lib/gtl/inlined_vector.h
index d6e5d9e..4079c66 100644
--- a/tensorflow/core/lib/gtl/inlined_vector.h
+++ b/tensorflow/core/lib/gtl/inlined_vector.h
@@ -283,9 +283,7 @@ class InlinedVector {
   static const size_t kFit = (kFit1 >= kSentinel) ? (kSentinel - 1) : kFit1;
 
   union {
-    unsigned char data[kSize];
-    // Force data to be aligned enough for a pointer.
-    T* unused_aligner;
+    alignas(T) unsigned char data[kSize];
   } u_;
 
   inline void InitRep() { u_.data[kSize - 1] = 0; }
