From 82d10fd0b4487350fb4915a1b4d13e9aba28fecf Mon Sep 17 00:00:00 2001
From: Christopher Feghali <christopher.feghali.xb@renesas.com>
Date: Wed, 22 Apr 2020 09:53:50 +0100
Subject: [PATCH] add cmake configuration for arm

Signed-off-by: Christopher Feghali <christopher.feghali.xb@renesas.com>
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9e1ac2dafe..dd7127b88f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,6 +29,8 @@ else()
 endif()
 
 set(CMAKE_INSTALL_MESSAGE NEVER)
+set(CMAKE_SYSTEM_PROCESSOR "armv7l")
+set(PROTOBUF_LIBRARY "${PROJECT_SOURCE_DIR}/../recipe-sysroot/usr/lib/libprotobuf.so")
 set(CMAKE_BUILD_TYPE "RelWithDebInfo")
 set(GLIBCXX_USE_CXX11_ABI "0")
 set(CMAKE_CROSSCOMPILING "1")
-- 
2.17.1

