From 350a719c0334f2a56db31e07407830d26cfa3d8d Mon Sep 17 00:00:00 2001
From: Gareth Williams <gareth.williams.jx@renesas.com>
Date: Mon, 7 Dec 2020 14:57:57 +0000
Subject: [PATCH] Select Tensorflow v2.3.1

Signed-off-by: Gareth Williams <gareth.williams.jx@renesas.com>
---
 workspace.bzl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/workspace.bzl b/workspace.bzl
index 5d05c69..df47a85 100644
--- a/workspace.bzl
+++ b/workspace.bzl
@@ -5,8 +5,8 @@ This module contains workspace definitions for building and using libedgetpu.
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 
-TENSORFLOW_COMMIT = "f394a768719a55b5c351ed1ecab2ec6f16f99dd4"
-TENSORFLOW_SHA256 = "cb286abee7ee9cf5c8701d85fcc88f0fd59e72492ec4f254156de486e3e905c1"
+TENSORFLOW_COMMIT = "fcc4b966f1265f466e82617020af93670141b009"
+TENSORFLOW_SHA256 = "ad41490904b0313c00f3729c7e2ba931b2b8d1e54ea1a859a4bcea3054d4fdc0"
 
 IO_BAZEL_RULES_CLOSURE_COMMIT = "308b05b2419edb5c8ee0471b67a40403df940149"
 IO_BAZEL_RULES_CLOSURE_SHA256 = "5b00383d08dd71f28503736db0500b6fb4dda47489ff5fc6bed42557c07c6ba9"
-- 
2.25.1

